react/features/authentication/constants.js

```diff
@@ -0,0 +1,9 @@
+/* global config, JitsiMeetJS, Promise */
+
+
+export const isTokenAuthEnabled
```

Comment:
I remember seeing this elsewhere during the review. Let's make sure it's in a single place, and as I mentioned earlier, use redux, or at least APP.store instead of directly accessing config.

Reply:
Created 2 functions that accept config parameters, renamed constants to functions and used APP.store.getState()['features/base/config'].